### PR TITLE
Cli: top

### PR DESCRIPTION
**Pull request by skotopes**
# What's new

- Cli: top command
- Furi: ThreadList container and thread enumeration routine.

# Verification 

- Check cli `top` command

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Comment by CookiePLMonster** [comment]
This PR should also update https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/AppManifests.md#parameters

> - **stack_size**: stack size in bytes to allocate for an application on its startup. Note that allocating a stack too small for an app to run will cause a system crash due to stack overflow, and allocating too much stack space will reduce usable heap memory size for apps to process data. _Note: you can use `ps` and `free` CLI commands to profile your app's memory usage._

**Review by DrZlo13 — DISMISSED** [review]
such wow, much cool

File: targets/f7/inc/FreeRTOSConfig.h [hunk 1]
```diff
@@ -30,12 +30,9 @@
 #define configMAX_TASK_NAME_LEN (32)
 
 /* Run-time stats - broken ATM, to be fixed */
```
Comment (hedger) [inline targets/f7/inc/FreeRTOSConfig.h]: not relevant anymore

File: furi/core/thread_list.c [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            FuriThreadListItemArray_remove(instance->items, it);
+            (void)FuriThreadListItemDict_erase(instance->search, (uint32_t)item->thread);
+            free(item);
+        } else {
+            uint32_t item_counter = item->counter_current - item->counter_previous;
+            if(item_counter && item->counter_previous && item->counter_current) {
+                item->cpu = (float)item_counter / (float)runtime_counter * 100.0f;
+                if(item->cpu > 200.0f) item->cpu = .0f;
+            } else {
+                item->cpu = 0.0f;
```
Comment (hedger) [inline furi/core/thread_list.c:104]: .0f 1 line above

**Review by gsurkov — DISMISSED** [review]

File: targets/f7/inc/FreeRTOSConfig.h (hunk 1 above)
Comment (skotopes) [reply targets/f7/inc/FreeRTOSConfig.h]: relevant because it is a part of TCB and default is 16.

**Review by hedger — APPROVED** [review]

**Pull request merged**